daemon/api/endpoints/flow.py

```diff
@@ -78,3 +79,18 @@ async def _status(
 @router.on_event('shutdown')
 def _shutdown():
     store.reset()
+
+
+@router.put(
+    path='/{id}',
+    summary='Run an update operation on the Flow object',
+    description='Types supported: "rolling_update" and "dump"',
```

Comment:
I am not sure if we should use a better naming, it is confusing to have `update` for a thing that can be `rolling_update` or `dump`.

Reply:
> to have `update` for a thing

What do you mean here? We are using the `PUT` method on the `/flows` endpoint. Is that what you mean by `update`? Not sure what the best alternative would be. I wouldn't add another endpoint, like `/flows/rolling_update` etc., as semantically it makes more sense to only have `/entity/ITS_ID`